scripts/effects/composure.lua

```diff
@@ -7,18 +7,22 @@
 local effectObject = {}
 
 effectObject.onEffectGain = function(target, effect)
-    local jpValue = target:getJobPointLevel(xi.jp.COMPOSURE_EFFECT)
+    local player = target:getMaster() or target
+    local mLevel = player:getMainLvl()
+    local jpValue = player:getJobPointLevel(xi.jp.COMPOSURE_EFFECT)
+    local accPower = math.floor(((24 * mLevel) + 74) / 49)
+    effect:setPower(accPower + jpValue)
 
-    target:addMod(xi.mod.ACC, 15 + jpValue)
+    player:addMod(xi.mod.ACC, effect:getPower())
```

Comment:
```lua
local power = math.floor((24 * target:getMainLvl() + 74) / 49) + target:getJobPointLevel(xi.jp.COMPOSURE_EFFECT)

effect:addMod(xi.mod.ACC, power)
```

Reply:
Thats all you need

Reply:
Thank you for the review!  I agree.  This approach is a leftover from a more complex implementation that adjusted the bonus accuracy dynamically using onTick, which I realized didn't reflect retail.  I'll clean it up as you've recommended here.